### ISSUE TITLE
bad text for global Docs home's link at the top

### ISSUE THREAD

**Issue by Erwan-le-Gall** [suitenumerique/docs#1259]
label: accessibility

The link "Docs" at the center top is labelled with "Logo Docs Docs".
The label must reflect the destination and the function of the link.
The fact that an pictogram or logo is included or used is unimportant.
The text should be "Retour à l’accueil général de Docs" (Back to the Docs' root home) or something similar.

<img width="183" height="124" alt="Image" src="https://github.com/user-attachments/assets/32326c88-23f8-45bc-ba1e-f45f72d63733" />

**Comment by Erwan-le-Gall** [comment]
the logo is decorative and should be removed from the accessibility tree.
solution : hide the logo (ie with <svg alt="" or <svg role="presentation") and remove the aria-label.
reference: https://css-tricks.com/accessible-svgs/

**Comment by Erwan-le-Gall** [comment]
the link's text must be explicit.
<a title="Retour à l’accueil de Docs" or <a alt="Retour à l’accueil de Docs"…
